src/Round.js

```diff
@@ -1,12 +1,11 @@
 import Puzzle from './Puzzle';
-import Wheel from './Wheel';
 import Turn from './Turn';
 
 class Round {
   constructor(puzzle, players, wheel) {
-    this.puzzle = new Puzzle(puzzle);
```

Comment:
Make sure to get rid of the parentheses after puzzle on this line.

Reply:
Good catch!